package main

import "testing"

func TestFail(t *testing.T) {
	t.Error("test failed")
}
